CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 cmake_minimum_required(VERSION 2.8)
 include(ExternalProject)
+set(CIME_ROOT "${CMAKE_CURRENT_SOURCE_DIR}")
 
 project(cime_tests Fortran C)
-
-list(APPEND CMAKE_MODULE_PATH ${CESM_CMAKE_MODULE_DIRECTORY})
-include(CESM_utils)
-find_package(NETCDF)
+list(APPEND CMAKE_MODULE_PATH "${CIME_ROOT}/externals/pio2/cmake")
```

Comment:
We have two different cmake directories and a lot of the code was both duplicated and diverging, for example findnetcdf.cmake was in both and broken in one.